babblesim/targets/edtthci/syscfg.yml

```diff
@@ -0,0 +1,10 @@
+syscfg.vals:
+  # EDTT requires 0x123456789ABC address for the first device
+  # and 0x456789ABCDEF for the second one
+  BLE_LL_PUBLIC_DEV_ADDR: 0x123456789ABC  # d=1
+  # BLE_LL_PUBLIC_DEV_ADDR: 0x456789ABCDEF # d=2
+
+  # Default task settings
+  OS_MAIN_STACK_SIZE: 8000
+  EDTT_HCI_LOG_FILE: ("hci_logs")
+  EDTT_HCI_LOGS: 1
```

Comment:
this does not seem to be defined anywhere, I guess it should be used instead of -DEDTT_HCI_LOGS ?